docs/book/src/cronjob-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -122,15 +124,18 @@ var _ = Describe("Manager", Ordered, func() {
 				"--clusterrole=project-metrics-reader",
 				fmt.Sprintf("--serviceaccount=%s:%s", namespace, serviceAccountName),
 			)
-			Expect(utils.Run(cmd)).Error().NotTo(HaveOccurred(), "Failed to create ClusterRoleBinding")
+			_, err := utils.Run(cmd)
+			Expect(err).Error().NotTo(HaveOccurred(), "Failed to create ClusterRoleBinding")
```

Comment:
These three Error() can also be removed.

```suggestion
			Expect(err).NotTo(HaveOccurred(), "Failed to create ClusterRoleBinding")
```